### DISCUSSION TITLE
"Remove All Below" Button in Saved Search Results List

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#129] [Ideas]
When creating a saved search, custom fields are always added below the stock fields for each search type.   Most of the time, I remove all these fields one by one.   I know that you can Remove All and then add the fields back in, but I think it would be helpful if there was a "Remove All Below" button that you could press, and all fields below below the selected field would be removed from the results.   I've attached a screenshot to illustrate this.

![2024-03-12 Remove All Below Button](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/160878598/094f4ea8-063f-481f-820f-82fba5ec4f1e)

**Comment by burkybang** [comment]
This'll be in the next release. Thanks for the idea!

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/9a06dd88-3998-4242-a68e-7cdcb8d00abd)

**Reply by dd-manderson** [reply]
Great!  this will be immensely helpful

**Reply by burkybang** [reply]
@dd-manderson The update is out now!

https://suiteadvanced.com/version-29_3_0
